frontend/src/utils/Firebase.ts

```diff
@@ -0,0 +1,14 @@
+import * as firebase from "firebase/app";
+
+const firebaseConfig = {
```

Comment:
Could you add a comment here saying it is okay to expose this as per this answer by a Firebase engineer: https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public